OpenKeychain/src/main/java/org/sufficientlysecure/keychain/util/Preferences.java

```diff
@@ -444,6 +446,40 @@ public boolean getExperimentalEnableKeybase() {
         return mSharedPreferences.getBoolean(Pref.EXPERIMENTAL_ENABLE_KEYBASE, false);
     }
 
+    // store comma separated masterKeyIds of all keys for which notification reminder is required
+    public void setBackupReminder(long masterKeyId) {
+        String reminders = mSharedPreferences.getString(Pref.BACKUP_REMINDERS,null);
+        reminders = (reminders == null) ? masterKeyId + "" : reminders + "," + masterKeyId;
+        SharedPreferences.Editor editor = mSharedPreferences.edit();
+        editor.putString(Pref.BACKUP_REMINDERS, reminders);
+        editor.apply();
+    }
+
+    public void removeBackupReminder(long masterKeyId) {
+        String reminders = mSharedPreferences.getString(Pref.BACKUP_REMINDERS,null);
+        if (reminders != null) {
+            List<String> masterKeyIds = new ArrayList<>(Arrays.asList(reminders.split(",")));
+            masterKeyIds.remove(masterKeyId+"");
```

Comment:
I need mSharedPreferences in this function which is non-static. So, I cannot make this static method. Is that what you mean or am I missing something??